--- SearchAndRescue-0.7.20/sar/Makefile.Linux.orig	Sun Apr 21 23:06:28 2002
+++ SearchAndRescue-0.7.20/sar/Makefile.Linux	Wed Jan 29 13:22:41 2003
@@ -117,9 +117,9 @@
 # Intel Pentium maximum speed
 CFLAGS = -D__USE_BSD \
          -DUSE_XSHM -DHAVE_MWMUTIL_H \
-         -DHAVE_LIBXPM -DJS_SUPPORT -DHAVE_Y2 \
-         -fomit-frame-pointer -funroll-loops -finline-functions \
-         -ffast-math -O6 -Wall -march=i586
+         -DHAVE_LIBXPM $(OPTFLAGS) \
+         -funroll-loops -finline-functions \
+         -ffast-math -Wall
 
 # Good efficiency for most generic situations
 #CFLAGS = -D__USE_BSD \
@@ -157,10 +157,8 @@
 #   farther above.
 #
 LIB = -lm \
-      -ljsw \
       -lSM -lICE -lX11 -lXext -lXi -lXmu -lGL -lGLU \
-      -lXpm \
-      -lY2
+      -lXpm 
 
 # Library Directories:
 #
@@ -179,7 +177,7 @@
 #   Each argument is of the format -I<dir> where <dir> is the full
 #   path to the directory.
 #
-INC =
+INC = -I/usr/X11R6/include
 
 
 # ########################################################################
